src/app.ts

```diff
@@ -12,15 +12,31 @@ interface TemperatureSummary {
   low: number
   average: number
 }
+let data: TemperatureReading[] = []
 
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  data = [...readings]
 }
 
 export function getTemperatureSummary(
   date: Date,
   city: string,
 ): TemperatureSummary | null {
-  //add here your code
-  return null
+  const temperatures = [] as number[]
+  let avg = 0
+  data.forEach((item) => {
+    if (item.city === city && item.time.getTime() === item.time.getTime()) {
```

Comment:
Why are you comparing the same values?
`item.time.getTime() === item.time.getTime()`